samples/excel/01-basics/multiple-property-set.yaml

```diff
@@ -0,0 +1,156 @@
+order: 3
+id: excel-multiple-property-set
+name: Multiple Property Set
+description: Setting multiple properties at once with the rich API object set() method.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#setup").click(setup);
+
+                $("#set-multiple-properties-with-object").click(setMultiplePropertiesWithObject);
```

Comment:
Please remove the indentation starting from this line. Will be easier to read the code in the narrow task pane.